packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -205,6 +213,12 @@ class PlanarFreehandROITool extends AnnotationTool {
     registerOpenContourEditLoop(this);
     registerOpenContourEndEditLoop(this);
     registerRenderMethods(this);
+
+    this._throttledCalculateCachedStats = throttle(
```

Comment:
can you add a configuration to have the cached stats on or off and I guess default to off?

Reply:
@sedghi , please take a look again. `calculateStats` configuration varaible has been added.